### PR TITLE
Release/v1.1

### LINKED ISSUE #30: Push Release branch to PRD

### PR DESCRIPTION
**Pull request by xFerchoVJ**

### CODE REVIEW

**Review by xFerchoVJ — COMMENTED** [review]
Ready to deploy

**Review by mivcm — APPROVED** [review]
Ready to deploy

**Review by mivcm — APPROVED** [review]
Readyyyy, everything it's okayyy!

**Review by mivcm — APPROVED** [review]

File: assets/css/main.css
Comment (mivcm) [inline assets/css/main.css:1]: ready!

**Review by mivcm — APPROVED** [review]

**Pull request merged**